test-server/fixtures/asset-paths/css-urls.html

```diff
@@ -1,19 +1,20 @@
 <!doctype html>
 <html>
-  <head>
-    <link rel="stylesheet" href="/css.urls.css" />
-
-    <style>
-      .with-rewritten-background-img-inline {
-        background-image: url('/img?url=fixtures/blue.png');
-      }
-    </style>
-  </head>
-  <body>
-    <div class="with-background-img"></div>
-    <div class="with-external-background-img"></div>
-    <div class="with-rewritten-background-img"></div>
-    <div class="with-rewritten-background-img-inline"></div>
-    <div style="background: url('/img?url=fixtures/pink.png'); no-repeat center;"></div>
-  </body>
+<head>
+  <link rel="stylesheet" href="/css.urls.css" />
+  <base href="/" />
```

Comment:
Adding a base element to this page to simulate the base ref includes localhost bug.

Reply:
That's a good call @andrewortwein - I agree it's better to create a separate page to cover the base logic. Will update.